packages/roosterjs-editor-api/lib/format/setFontSize.ts

```diff
@@ -1,4 +1,6 @@
 import applyInlineStyle from '../utils/applyInlineStyle';
+import { applyStyleToListItems } from '../utils/applyStyleToListItems';
+import { findClosestElementTypeAncestor } from 'roosterjs-editor-dom';

```

Comment:
you can also use findClosestElementAncestor here. It has a "selector" parameter:

export default function findClosestElementAncestor(
    node: Node,
    root?: Node,
    selector?: string
): HTMLElement {

Reply:
And these 3 files are almost the same. Consider add a common wrapper for them to do the duplicated part work

Reply:
I see, I will remove the added function and use this instead.
Adding a wrapper too